packages/directory/addon/templates/components/dir-item-name-cell.hbs

```diff
@@ -1,8 +1,8 @@
 {{!-- Copyright 2018, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. --}}
 {{#link-to itemLink (get value 'id') classNames='dir-item-name-cell'}}
-  {{navi-icon iconClass classNames=(concat 'dir-item-name-cell__icon dir-icon__' type)}}
+  {{dir-icon iconClass}}
```

Comment:
Can we have our own classNames here so that they can be used in tests?
We don't want to use the font-awesome class names in case we move away from font-awesome in the future, we do not want to update a whole bunch of tests.

Reply:
I made this wrapper class because `fa-icon` component wasn't updating the passed in classNames correctly. At this point, we can't rely on our own classNames to apply styling to these icons.